### DISCUSSION TITLE
An error happened: Failed to fetch Snap "local:http://localhost:8080": Invalid Snap manifest: manifest shasum does not match computed shasum..

### DISCUSSION THREAD

**Discussion by enu-kuro** [MetaMask/snaps#1346] [Q&A] [answered]
How can I fix the error? 
I didn't upgrade Flask; I just edited the code in the snap and got the error. 
It seems that some developers have started experiencing the same error in the last few hours.

https://github.com/MetaMask/snaps-monorepo/discussions/1198#discussioncomment-5597643

**Comment by renzhaozhao** [comment]
I have some error, my Flask version is 10.28.2.

**Reply by yywana** [reply]
me too. still get that error
An error happened: Failed to fetch Snap "local:http://localhost:8080": Invalid Snap manifest: manifest shasum does not match computed shasum..

**Comment by FrederikBolding** [comment, marked as answer]
Hey there! Please update your `snaps-cli` to at least `^0.30.0` and rebuild your snap. That should create a new checksum that is compatible with Flask `10.28.2`. Release notes for the new version of Flask are on the way!

**Reply by FrederikBolding** [reply]
Announcement here: https://github.com/MetaMask/snaps-monorepo/discussions/1348

**Reply by ritave** [reply]
@renzhaozhao
